examples/layers/rendering/custom_coordinate_systems.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
+// @dart = 2.10
```

Comment:
Does this turn on null-safety, but in mixed mode for `examples/layers`?

Reply:
This allows us to use null safety syntax in this file (i.e. the `!` and `?`).